File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/GetParkingZoneDataResponse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by lattis on 24/05/17.
 */

public class GetParkingZoneDataResponse {
    @SerializedName("parking_area_id")
    private int parking_area_id;
    @SerializedName("name")
    private String zone_Name;
    @SerializedName("type")
    private String type;
    @SerializedName("zone")
    private String zone;

    @SerializedName("geometry")
    List<ParkingZoneGeometryResponse> parkingZoneGeometry;
    @SerializedName("operator_id")
    private int operatorID;
    @SerializedName("customer_id")
    private int customerID;
    @SerializedName("fleet_id")
    private int fleetID;


    public List<ParkingZoneGeometryResponse> getParkingZoneGeometry() {
        return parkingZoneGeometry;
    }

    public int getParking_area_id() {
        return parking_area_id;
    }

    public String getZone_Name() {
        return zone_Name;
    }

    public String getType() {
        return type;
    }

    public String getZone() {
        return zone;
    }

    public int getOperatorID() {
        return operatorID;
    }

    public int getCustomerID() {
        return customerID;
    }

    public int getFleetID() {
        return fleetID;
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/GetParkingZoneDataResponse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName

class GetParkingZoneDataResponse {
    @SerializedName("parking_area_id")
    val parking_area_id = 0

    @SerializedName("name")
    val zone_Name: String? = null

    @SerializedName("type")
    val type: String? = null

    @SerializedName("zone")
    val zone: String? = null

    @SerializedName("geometry")
    var parkingZoneGeometry: List<ParkingZoneGeometryResponse>? = null

    @SerializedName("operator_id")
    val operatorID = 0

    @SerializedName("customer_id")
    val customerID = 0

    @SerializedName("fleet_id")
    val fleetID = 0

}